### DISCUSSION TITLE
Add support for touchscreen based devices

### DISCUSSION THREAD

**Discussion by jonch070** [helgoboss/helgobox#1341] [Ideas]
touchable/lemur style

**Comment by helgoboss** [comment]
You can already do that, ReaLearn has OSC support.

**Comment by helgoboss** [comment]
One more note. I assume that you are talking about Playtime here.

- If you have a touch screen (Windows), Playtime should already work with that. The usability is limited because some UI elements are not optimized for touch screen usage, but if you increase the sizes in the settings, it should be sort of usable.
- At some point, I'm planning to release the Helgobox App as Android and hopefully also iOS app, which will probably also include some touch screen optimizations.
